helix-term/src/config.rs

```diff
@@ -3,6 +3,7 @@ use serde::Deserialize;
 use crate::keymap::Keymaps;
 
 #[derive(Debug, Default, Clone, PartialEq, Deserialize)]
+#[serde(deny_unknown_fields)]
```

Comment:
You probably want this on the editor config too: https://github.com/helix-editor/helix/blob/2c1313c0648977540c395de584b4293c1909cbf7/helix-view/src/editor.rs#L38